### ISSUE TITLE
缺少 net.eval() and net.train()

### ISSUE THREAD

**Issue by CarryHJR** [bubbliiiing/yolo3-pytorch#2]
网络中用了batchnorm，应该在eval时加上net.eva()

**Comment by bubbliiiing** [comment]
我看没有影响网络的预测效果所以没加，这个问题大么

**Comment by CarryHJR** [comment]
不算大，我之前相关竞赛中不加net.eval()会有一个点的损失

**Comment by bubbliiiing** [comment]
emm其实我不太清楚，我对pytorch的机制不太了解，因为我有加过，甚至导致检测效果下降

**Comment by CarryHJR** [comment]
batchnorm和dropout对于train 和 val两种状态中不一样，理论上肯定要加，不过都是玄学，let it go
